CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## Unreleased
```

Comment:
Thank you for this, but there is no need to update the changelog! ;)

Reply:
Okay :D I'll remove it. Thanks